src/EventMetadata.php

```diff
@@ -27,6 +27,9 @@ public function __construct(
         array $customFields,
         int $timestampMillis = null
     ) {
+        if ($timestampMillis == null) {
```

Comment:
We should write a test that asserts that upon construction the `timestampMillis` will not be null.

Reply:
I had to add a getter for `timestampMillis` to assert that it's not null, but I went ahead and did that.